src/docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG DOCKER_TAG=latest
-ARG GRAPHISTRY_FORGE_BASE_VERSION=latest
+ARG DOCKER_TAG=latest-11.4
+ARG GRAPHISTRY_FORGE_BASE_VERSION=latest-11.4
```

Comment:
should be something like `v2.39.12-11.4`, corresponding to something at https://hub.docker.com/r/graphistry/graphistry-forge-base/tags